src/main/paradox/docs/api/iam/iam-acls-api.md

```diff
@@ -1 +1,206 @@
-# Access Control List
\ No newline at end of file
+# Access Control Lists
+
+Access Control Lists are rooted in the `/v1/acls` collection.
+
+An ACL defines the applications' data access restriction using the following three parameters:
+         
+- permission: the value used to limit application's access to information.
+- identity: the way to identify individuals.
+- path: the address where to apply the restrictions. Examples of paths are: `/`, `/myorg` or `/myorg/myproject`
+
+Any resources in the system might be protected using an **access token**, provided by the HTTP header `Authorization: Bearer {access_token}`. Visit @ref:[Authentication](./authentication.md) in order to learn more about how to retrieve an access token.
+
+## ACLs Hierarchy
+
+It is important to know that ACLs are represented in a tree-like structure depending on their path. Imagine the following scenario:
+![Resources tree](../assets/permissions-tree.png "Permissions tree")
+
+Each block is identified by a path that contains a list of permissions for a certain identity (identities are color code divided). There is a special set of permissions which restrict the use of the ACLs API
+
+- **acls/read** - an auth. token containing an identity with this permission is allowed to fetch a collection of ACL from any other identity.
+- **acls/write** - an auth. token containing an identity with this permission is allowed to perform the call to the following endpoints: [cerate ACLs](#create-acls), [replace ACLs](#replace-acls), [subtract ACLs](#subtract-acls), [append ACLs](#append-acls) and [delete ACLs](#delete-acls).
+
+Those permissions need to be present in the current `{address}` where the API interaction occurs or in any parent address. In other words, they are inherited.
+
+Let's clarify this concept with an example from the previous diagram. `identity 1` could call the [create ACLs](#create-acls) endpoint on any `{address}` while `identity 2` could only call the same endpoint for any address child of `/myorg` (like `/myorg/myproj`). At the same time, `identity 3` could not perform any of the write operations.
```

Comment:
I don't see 'identity 1' having 'acls/write' permission from the figure. It has acls/read permission allowing to fetch acls granted to any identity.